README.md

```diff
@@ -4,6 +4,15 @@
 # telemetry-streaming
 Spark Streaming ETL jobs for Mozilla Telemetry
 
+This service currently contains jobs that aggregate error aggregate data
```

Comment:
You can drop the second `aggregate` on this line

Reply:
lol yes